### ISSUE TITLE
How to train the diffusart model in a new customized datasets

### ISSUE THREAD

**Issue by HONGJINLYU** [hernan0930/Diffusart-CVPRW#1]
quite a great work in diffusion-based colorization research field.
Many thanks for your released code.

But could you please provide some guidance on training the model with a new dataset. Here are the details:

1. Preparing the Training Dataset
how to prepare the training dataset

2. Configuring Training Parameters
how to config the parameters during the training process
   - **Model Architecture:
   - **Loss Function:
   - **Optimization Algorithm:
   - **Training Batch Size and Epochs:
   - **Learning Rate Schedule:

3. Starting the Training Process
   Once I have prepared my dataset and configured the training parameters, could you please tell me how I can start the training process using a command-line interface or a training script provided with the codebase. 

I hope you could kindly provide some guidance help other researchers  get started with training the model on their customized dataset. 
 and I'm looking forward to hearing about your progress!
